packages/editor-ui/src/components/mixins/showMessage.ts

```diff
@@ -33,11 +52,34 @@ export const showMessage = mixins(externalHooks).extend({
 				};
 			}
 
+			if (config.onLinkClick) {
+				const onLinkClick = (e: MouseEvent) => {
+					if (e && e.target && config.onLinkClick && isChildOf(notification.$el, e.target as Element)) {
```

Comment:
isn't this checked 2 lines above? Or is TS yelling then?

```suggestion
					if (e && e.target && isChildOf(notification.$el, e.target as Element)) {
```

Reply:
TS yelling 😊